manual/manual/cmds/top.etex

```diff
@@ -72,13 +72,13 @@ latter is read and executed as described below.
 
 The toplevel system does not perform line editing, but it can
 easily be used in conjunction with an external line editor such as
-"ledit", "ocaml2" or "rlwrap"
+"ledit", or "rlwrap". There is also an improved toplevel, "utop",
+available in
 \begin{latexonly}
-(see the Caml Hump "http://caml.inria.fr/humps/index_framed_caml.html").
+the opam repository \url{https://opam.ocaml.org}.
 \end{latexonly}
 \begin{htmlonly}
-(see the
-\ahref{http://caml.inria.fr/humps/index\_framed\_caml.html}{Caml Hump}).
+the \ahref{http://https://opam.ocaml.org}{opam repository}.
```

Comment:
I don't think that the mention of the opam repository makes much sense here in context. If people have an idea of how to install `utop`, this is useless, and if they don't it isn't the right place to explain it.

Reply:
Should I just remove the link with the idea that the link to Caml Hump is more confusing than informative nowadays?

Reply:
Yes, that sounds reasonable.

Reply:
Done.